pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import "./deleteModal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash, faTimes } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  resourceDelete: () => void;
+  resourceId: string;
+  optionalMessage?: JSX.Element;
+}
+
+const DeleteModal: React.FC<DeleteModalProps> = ({ resourceDelete, resourceId, optionalMessage }) => {
+  return (
+    <div className="DeleteModal">
+      <Modal
+        title="Warning!"
+        id={`deleteModal${resourceId}`}
+        body={function () {
```

Comment:
You can shorthand this notation making it easier to read and maintain. 

Just like on line `8` where we're declaring `resourceDelete: () => void` which basically means a function that returns a void. We can do the following here:

```
() => <> component here </>
```

This works a follwed:

```
() => {
  <>component</>
  // this is not instantly returned, because it's within brackets ({})
}

() => <>component</> // this IS instantly returned, because it is NOT in brackets

() => (
  <>component</> // this IS ALSO instantly returned, because it is NOT in brackets
)
```